mleap-spark/src/main/resources/spark-demo.csv

```diff
@@ -0,0 +1,7 @@
+test_string,test_double
```

Comment:
We probably shouldn't include this with the `mleap-spark` module. Can we create a folder next to `mleap-spark` called `examples` and add it here?

Reply:
Yep, good point, I intended to put it under test/resources. I've created the examples folder as you suggested instead.